### ISSUE TITLE
Remove babel-messages and inline the usages

### ISSUE THREAD

**Issue by hzoo** [babel/babel#6347]
TL;DR remove `babel-messages` package and inline it's contents into where we are using it.

---

I was looking at https://github.com/babel/babel/tree/master/packages/babel-messages and it's usage in the codebase. It seems like it's only used in 7 files, and doesn't seem to be any sharing of the messages across the codebase so it seems like it's not necessary to have anymore.

- Read our [contributing.md](https://github.com/babel/babel/blob/master/CONTRIBUTING.md) guide!
- Find all files that import `"babel-messages"` like `import * as messages from 'babel-messages';`
- [ ] Remove the import, and then inline the message itself

So like for `option-manager.js` which has 1 usage: 

https://github.com/babel/babel/blob/048a5b80214520c4123728a8c124ef0ad9e76511/packages/babel-core/src/config/option-manager.js#L339

Just replace the `messages.get("pluginInvalidProperty", descriptor.alias, key)` with what the message should say. In this case it would be:

```
`Plugin ${descriptor.alias} provided an invalid property of ${key}`
```

If there are arguments (2 in this case), you will need to use a template string and substitute the $1 and $2 with template literal braces to put the variables.

- [ ] remove the whole folder `packages/babel-messages` and make sure it's not referenced anywhere else
- [ ] run tests, see if they all pass!

**Comment by rouzbeh84** [comment]
oh nice! @hzoo i can handle this 👍 .

**Comment by existentialism** [comment]
@rouzbeh84 join us on slack if you have any questions!

**Comment by JeromeFitz** [comment]
I know this issue is claimed, but thought it would be a good one to at test out how to contribute in the future.

@rouzbeh84 - My only goal with my referenced commit/ branch is to track against your future one and see how they line up.

**Comment by hzoo** [comment]
Totally @jeromefitz! I was thinking before that it is unfortunate that issues especially beginner friendly ones are only claimed or done by one person (whoever happens to see it first). I would definitely encourage everyone to try a PR and compare solutions if not just try it out to get a feel of contributing in the future.

I will try to encourage this much more in the future, thanks for re-affirming my intuition!

**Comment by JeromeFitz** [comment]
*Wasn't enough room in Twitter for this question.* 😀️

**In dealing with prettier and multi-line strings**

Should we keep them uniform if they have a template string:
```javascript
throw new Error(
  `You passed \`traverse()\` a visitor object with the property ` +
    `${nodeType} that has the invalid property ${visitorKey}`,
);
```
Or literal based on what the line contains:
```javascript
throw new Error(
  "You passed `traverse()` a visitor object with the property " +
    `${nodeType} that has the invalid property ${visitorKey}`,
);
```

**Comment by rouzbeh84** [comment]
@JeromeFitz i've created PR #6352 with what I had as well as a checklist of files affected as far as I could tell. Feel free to pick up where I left off or let me know which one's you'd like to handle and I can help on the remaining. 

If I did these right myself (which now looking at your question I think I didn't so thank you! fix/rebase pushed to PR). Anyways, I think you'd be fine in doing the following:
```
throw new Error(
  `You passed \`traverse()\` a visitor object with the property ${nodeType} 
   that has the invalid property ${visitorKey}`,
);
```
I do hope someone confirms this as I had to break lines as well adhere to precommit checks so hopefully it's not a problem?

**Comment by Andarist** [comment]
@rouzbeh84 
the single literal shouldnt be used (imho) because the second line will get unfortunately unnaturally indented.

@JeromeFitz 
I would probably chose the latter, but any of them is fine really - if only allowed by prettier and eslint

**Comment by existentialism** [comment]
Yeah, either is fine (agree with @Andarist, I'd probably do the latter), and both should pass lint and be preserved by prettier.

**Comment by JeromeFitz** [comment]
Cool @rouzbeh84, bringing down your PR now and will close out the rest of the references. Thanks @Andarist (& @existentialism), we'll preserve prettier + lint based on your feedback. 👍️
